baybe/kernels/priors/gamma.py

```diff
@@ -0,0 +1,16 @@
+"""A Gamma prior parameterized by concentration and rate."""
+
+from attrs import define, field
+from attrs.validators import gt
+
+from baybe.kernels.priors.base import Prior
+
+
+@define
+class GammaPrior(Prior):
+    """A Gamma prior parameterized by concentration and rate."""
+
+    concentration: float = field(validator=gt(0.0))
```

Comment:
I was not sure which values to choose here as defaults since we basically choose between 4 different sets of default values ourselves currently. So which should be chosen?

Reply:
i see, I was assuming the Gpytorch kernels have defaults, but it seems they dont, guess then we dont have to bother with them although I always find it nice
isnt there a "general" setting of these settings that just makes an extremely broad prior?

Reply:
(and of course, we can always add defaults later on if we think it's reasonable)

Reply:
I see it as part of the GP refactoring PR that we assigned to me because you basically can not solve it here. So will close the thread. @Scienfitz: please unresolve if necessary